Admin/CommentAdmin.php

```diff
@@ -146,9 +146,9 @@ public function postUpdate($object)
     }
 
     /**
-     * @param ManagerInterface $commentManager
+     * @param CommentManagerInterface $commentManager
      */
-    public function setCommentManager(ManagerInterface $commentManager)
+    public function setCommentManager(CommentManagerInterface $commentManager)
```

Comment:
@Soullivaneuh Okay, or should I split this in two seperate PRs: phpdoc and interface changes?